app-arch/pet2tgz/pet2tgz-0.3-r1.ebuild

```diff
@@ -10,6 +10,5 @@ SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 DOCS="AUTHORS"
```

Comment:
There is one last thing. You can drop that, since with ```EAPI=7``` several default docs are automatically installed.
See https://devmanual.gentoo.org/ebuild-writing/functions/src_install/index.html for future reference.